src/helpers/routes/routes.js

```diff
@@ -35,6 +35,12 @@ class APIRoutes {
   static veteransMilitaryBranch() {
     return APIRoutes.create('/veterans/get_military_branch');
   }
+  static veteransPhoneNumber() {
+    return APIRoutes.create('/veterans/get_phone_number');
+  }
+  static veteransAddress() {
+    return APIRoutes.create('/veterans/get_address');
+  }
```

Comment:
I don't think we need these routes. I believe we needed `veteransMilitaryBranch` in order to get a list of all the military branches that are choosable from the API. In this case a phone number and address would be properties of a single veteran object so it would be returned as a field in one veteran object when `veteransPath` is called to retrieve all the veterans.